api/origin_trials_api.py

```diff
@@ -136,10 +146,9 @@ def do_post(self, **kwargs):
     if ot_stage is None:
       self.abort(404, msg=f'Stage {ot_stage_id} not found')
 
-    # Check that user has permission to edit the feature
-    # associated with the origin trial.
-    redirect_resp = permissions.validate_feature_edit_permission(
-        self, feature_id)
+    # Check that user has permission to edit the feature associated
+    # with the origin trial, or has a @google or @chromium account.
+    redirect_resp = self.check_post_permissions(feature_id)
```

Comment:
Should we catch the exception and return an Auth error?

Reply:
We don't explicitly raise an exception, instead we do self.abort(403) which calls a flask function that raises an exception that is caught by flask.  The 403 code gets passed back to the browser.

For the case where the user is signed out, I suppose that you are correct.   While a UI request can respond with a 302 redirect, an API request should respond with 401.   However, that seems beyond the scope of this PR.  I'll add a TODO to consider that as part of converting to using OpenAPI.